### ISSUE TITLE
Color picker

### ISSUE THREAD

**Issue by tonamphong** [WPTT/control-color-alpha#5]
Hi there,
I followed the tutorial to implement the color alpha feature, it's working. However, the color picker still appear after choosing the color (It should close the color picker after clicking outside the box, right ?). Did I miss something ?
It's still appear like this https://make.wordpress.org/themes/files/2019/12/Screenshot_2019-12-14-Customize-two-wordpress-test-%E2%80%93-Just-another-WordPress-site.png

**Comment by aristath** [comment]
Yes, that is correct. The control currently doesn't collapse. You didn't miss anything, it was intentionally built like that.
